### PR TITLE
#166187843 Integrate coveralls code coverage service

### PR DESCRIPTION
**Pull request by alexxsanya**
#### What does this PR do?
- Integrate coveralls code coverage service

#### Description of Task to be completed?

- add the script to the travis file to setup & run coverall
- add the coverage badge in the ReadMe file

#### How should this be manually tested?
- Click [here](https://coveralls.io/github/andela/ah-backend-realers?branch=ch-integrate_coveralls-166187850) to see the evidence of coverall being setup successfully

#### Any background context you want to provide?
- The project currently has no coverall setup to determine how much testable code is in the codebase, so this PR sets it up.

#### What are the relevant pivotal tracker stories?

- Story ID - [166187843](https://www.pivotaltracker.com/story/show/166187843)

### CODE REVIEW

**Review by neelxie — COMMENTED** [review]
@alexxsanya kindly please enlighten me about the particular script, you mean in the `description` section of this PR and I am wondering why the badge shows unknown on your branch as shown below in the screenshot, thank you.
Screenshot 2019-05-28 at 16 12 10

**Comment by alexxsanya** [comment]
> @alexxsanya kindly please enlighten me about the particular script, you mean in the `description` section of this PR and I am wondering why the badge shows unknown on your branch as shown below in the screenshot, thank you.
> Screenshot 2019-05-28 at 16 12 10

@neelxie kindly note that the badge in the README file is pointing to the `develop` branch which currently doesn't have coverall integration but once the merge is done it will get a correct values. That's the reason why in the procedures I provided a link to checkout to verify that coverall is integrated into the `ch-integrate_coveralls-166187850` branch.

**Review by Emmanuelaaron — APPROVED** [review]
Great Job @alexxsanya , I confirm it is set up successfully on clicking the link provided

**Review by jennizalwango — COMMENTED** [review]
Well @alexxsanya, So apparently the lines of code added to, look good and l have tested it as explained in the description and l wasn't disappointed. so  l think your good to go ahead

**Review by jennizalwango — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-[![Build Status](https://travis-ci.org/andela/ah-backend-realers.svg?branch=ch-intergrate_Travis-166187847)](https://travis-ci.org/andela/ah-backend-realers)
-
+[![Build Status](https://travis-ci.org/andela/ah-backend-realers.svg?branch=ch-intergrate_Travis-166187847)](https://travis-ci.org/andela/ah-backend-realers) [![Coverage Status](https://coveralls.io/repos/github/andela/ah-backend-realers/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-backend-realers?branch=develop)
```
Comment (HabibSentongo) [inline README.md:1]: This is great @alexxsanya you went beyond the task and added the Coverage badge to the `readme.md`

**Review by HabibSentongo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (alexxsanya) [reply README.md:1]: thanks @HabibSentongo

**Review by neelxie — APPROVED** [review]

**Pull request merged**